### PR TITLE
🍿 [Portfolio] Position components

### PR DESCRIPTION
**Pull request by marcinciarka**
🍿 [Portfolio] Position components

### CODE REVIEW

File: components/portfolio/positions/PortfolioPositionAutomationIcons.tsx
Comment (piotrkonowrocki) [inline components/portfolio/positions/PortfolioPositionAutomationIcons.tsx:1]: I think I saw some designs with automation icons in Product Hub in the table, maybe this component could be places inside generic components without direct connection to Portfolio in its name.

File: components/GenericSelect.tsx [hunk 1]
```diff
@@ -40,6 +40,7 @@ export interface GenericSelectProps {
   options: GenericSelectOption[]
   placeholder?: string
   wrapperSx?: ThemeUIStyleObject
+  iconPosition?: 'left' | 'right'
```
Comment (piotrkonowrocki) [inline components/GenericSelect.tsx:43]: Imo we should use select as is, as we have it implemented right now without creating another variant of it for the sake of consistency.

**Review by piotrkonowrocki — CHANGES_REQUESTED** [review]
Please also update imports to absolute path in all of the files.

File: components/GenericSelect.tsx (hunk 1 above)
Comment (marcinciarka) [reply components/GenericSelect.tsx]: Removed

**Pull request merged**